boxes/scripts/steps/sandbox/install.js

```diff
@@ -37,6 +37,14 @@ const runPty = async (command, { success: exitSuccess, error }) => {
   }
 };
 
+async function installSandbox() {
+  await runPty("echo y | bash -i <(curl -s install.aztec.network); exit\n", {
```

Comment:
Might prefer using `yes` here too?

Reply:
ofc I didn't even know about the yes option